### PR TITLE
Added Json Reporter

### PR DESCRIPTION
**Pull request by laurentmuller**

### CODE REVIEW

File: src/Reporter/JsonReporter.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                'column' => $violation->getColumn(),
+                'severity' => $violation->getSeverity(),
+                'type' => strtolower($violation->getSeverityAsString()),
+                'message' => $violation->getReason(),
+            ];
+        }
+
+        $output->writeln(json_encode($result, JSON_PRETTY_PRINT));
+    }
+}
```
Comment (Ciloe) [inline src/Reporter/JsonReporter.php:40]: Add end of file please

**Review by Ciloe — CHANGES_REQUESTED** [review]
Can you add tests for this class ? Thanks for your commit.

File: src/Reporter/JsonReporter.php (hunk 1 above)
Comment (laurentmuller) [reply src/Reporter/JsonReporter.php:40]: What do You mind with "add end of file"?

Comment (Ciloe) [reply src/Reporter/JsonReporter.php:40]: If you go on `Files changes` tab, you will see on the end of file a litle logo red. On mouse over it said `Non new line at the end of file`. Just add a empty line at the end of file will fix it 😃

**Comment by Ciloe** [comment]
I see on your class that you have an if when the file is not the same at the global file. Can you test it ? After it, will be good 👍 

Edit
====

I have read more effeciently, just test with an error in two differente file will be good !

**Review by OwlyCode — APPROVED** [review]

**Comment by OwlyCode** [comment]
Thank you @laurentmuller

**Pull request merged**